frontend/src/ui/Blocks/Series.tsx

```diff
@@ -234,27 +234,40 @@ const VideoGrid: React.FC<React.PropsWithChildren> = ({ children }) => (
 const UpcomingEventsGrid: React.FC<React.PropsWithChildren> = ({ children }) => {
     const { t } = useTranslation();
 
-    return <details css={{
-        backgroundColor: "var(--grey86)",
-        borderRadius: 4,
-        "> summary": {
-            color: "var(--grey20)",
-            cursor: "pointer",
-            fontSize: 14,
-            padding: "6px 12px",
-        },
-        "> hr": {
-            margin: "0 12px",
-        },
-    }}>
-        <summary>
-            {t("series.upcoming-live-streams", { count: Children.count(children) })}
-        </summary>
-        <hr />
-        <VideoGrid>
-            {children}
-        </VideoGrid>
-    </details>;
+    return (
+        <details css={{
+            backgroundColor: "var(--grey86)",
+            borderRadius: 4,
+            marginBottom: "8px",
+            "> summary": {
+                color: "var(--grey20)",
+                cursor: "pointer",
+                fontSize: 14,
+                padding: "6px 12px",
+                "> span": {
+                    marginLeft: "2px",
+                },
+                ":hover, :focus-visible": {
+                    backgroundColor: "var(--grey80)",
+                    color: "var(--accent-color-darker)",
```

Comment:
Not sure about the text color change. I would leave it as is or make it just black. I don't really see the advantage of the accent color here and it potentially makes contrast worse.